SmartDeviceLink/SDLLifecycleManager.m

```diff
@@ -219,6 +222,9 @@ - (void)didEnterStateStarted {
                                             tcpPort:@(self.configuration.lifecycleConfig.tcpDebugPort).stringValue
                           secondaryTransportManager:self.secondaryTransportManager];
     } else {
+        // start a background task so a session can be established even when the app is backgrounded.
+        [self sdl_backgroundTaskStart];
```

Comment:
I think this should always run when we are running SDL, not just in iAP.

Reply:
Fixed